lib/features/create-append-anything/CreateAppendEditorActions.js

```diff
@@ -0,0 +1,44 @@
+import { assign } from 'min-dash';
+
+/**
+ * Registers and executes BPMN specific editor actions.
+ *
+ * @param {Injector} injector
+ */
+export default function CreateAppendEditorActions(injector, editorActions) {
```

Comment:
Consider to make this optionally depend on `editorActions`.

You'll otherwise not be able to properly compose it.

Alternative (but maybe overkill):

* add `create-append-anything/editor-actions` and `keyboard` modules